packages/components/src/range-control/index.js

```diff
@@ -30,13 +30,16 @@ function RangeControl( {
 	min,
 	max,
 	setState,
+	initialValue,
 	...props
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const currentInputValue = currentInput === null ? value : currentInput;
+	const initialInputValue = ! ( initialValue && min <= initialValue && max >= initialValue ) ?
```

Comment:
It would read easier if the check would start with a positive condition. 
```suggestion
	const initialInputValue = ( initialPosition && initialPosition >= min && initialPosition <= max ) ? parseFloat( initialPosition ) : undefined;
```

Reply:
So I used initialValue instead of initialPosition for reason above.